### PR TITLE
User following 2

### PR DESCRIPTION
**Pull request by LinhBM**

### CODE REVIEW

**Comment by LinhBM** [comment]
screenshot from 2017-05-30 17-28-45

File: app/views/users/show_follow.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<% provide(:title, @title) %>
```
Comment (dat-hedspi) [inline app/views/users/show_follow.html.erb]: bỏ ()

File: app/views/users/show_follow.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= link_to image_tag(@user.image.url(:small)), @user.image.url %>
+      <h1><%= @user.username %></h1>
+      <span><%= link_to "view my profile", @user %></span>
+    </section>
+    <section class="stats">
+      <%= render 'shared/stats' %>
```
Comment (dat-hedspi) [inline app/views/users/show_follow.html.erb]: double quote

File: app/views/users/show_follow.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<% provide :title, @title %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      <%= link_to image_tag(@user.image.url(:small)), @user.image.url %>
+      <h1><%= @user.username %></h1>
+      <span><%= link_to "view my profile", @user %></span>
```
Comment (dat-hedspi) [inline app/views/users/show_follow.html.erb]: I18n

**Pull request merged**